src/lib.rs

```diff
@@ -0,0 +1,90 @@
+/*
+ *   Copyright (c) 2020 
+ *   All rights reserved.
+ */
+
+pub mod read_create {
+    extern crate calamine;
+    use calamine::{open_workbook, Xlsx};
+    use std::fs::File;
+    use std::io::BufReader;
+
+
+pub fn create_file(input_file:&str) -> File{
+
+    let file_handle = File::create(input_file).expect("unable to create a output file");
+    return file_handle;
```

Comment:
use return keyword only when u are explicitly or forcefully return something here instead of `return file_handle;`,  `file_handle` is worth full